server/public/model/utils.go

```diff
@@ -481,15 +481,28 @@ func ArrayToJSON(objmap []string) string {
 	return string(b)
 }
 
-func ArrayFromJSON(data io.Reader) []string {
-	var objmap []string
+func ArrayFromJSON(data io.Reader, maxBytes int64) []string {
```

Comment:
Debated about returning error, rather than empty array (as current implementation does). If we do, then all the utility functions should be updated. Thoughts?

Reply:
It's always good to return the error and bubble it up. We have an opportunity to do the right thing here. I'd say we should take it. Unfortunately, this means updating all those functions :(